### ISSUE TITLE
Unable to have multiple CesiumWidets on the same page in 3d-tiles branch

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#5226]
In this example, I'm creating a new `CesiumWidget` for each of 4 containers.  However, the globe doesn't render in two of them.  This example works in master.

``` html
<style>
    @import url(../templates/bucket.css);
</style>
<table id="cesiumTable"></table>
<div id="loadingOverlay"><h1>Loading...</h1></div>
<div id="toolbar"></div>
```

``` javascript
var widgets = [];
var rows = 2;
var cols = 2;
var table = document.getElementById('cesiumTable');
table.innerHTML = '';
for (var i = 0; i < rows; i++) {
    var tr = document.createElement('tr');
    table.appendChild(tr);
    for (var j = 0; j < cols; j++) {
        var td = document.createElement('td');
        tr.appendChild(td);
        var div = document.createElement('div');
        div.style.height = '200px';
        div.style.width = '400px';
        td.appendChild(div);
        widgets.push(new Cesium.CesiumWidget(div));
    }
}
```

**Comment by lilleyse** [comment]
This is a bug related to the `RequestScheduler`. The scheduler budgets are stored at global scope but the budgets are reset in `Scene`, which would get called four times and mess up the budgets. If there is a good way to call `RequestScheduler.resetBudgets` just once per frame outside of any cesium widgets, that would be the fix.

**Comment by lilleyse** [comment]
Also the request scheduler architecture has potential to change with @austinEng's HTTP2 testing.

**Comment by pjcozzi** [comment]
This should be fixed before `3d-tiles` is merged into `master`.

**Comment by pjcozzi** [comment]
Already fixed in #5308?

**Comment by mramato** [comment]
Yes
